bundle/manifests/registry-operator.clusterserviceversion.yaml

```diff
@@ -615,7 +448,7 @@ spec:
                 - --leader-elect
                 command:
                 - /manager
-                image: quay.io/devfile/registry-operator:v0.1.3
+                image: quay.io/devfile/registry-operator:next
```

Comment:
Out of curiosity, I see the `spec.version` field inside the `config/manifests/bases/registry-operator.clusterserviceversion.yaml`. Should we also update this field as `next`?

Reply:
This field would be updated to the new version cut `v0.1.4` in a separate release PR, we can leave it `next` under the main branch.